deye_mqtt.py

```diff
@@ -32,11 +33,15 @@ def __init__(self, config: DeyeConfig):
             client_id=f'deye-inverter-{config.logger.serial_number}', reconnect_on_failure=True)
         self.__mqtt_client.enable_logger()
         if config.mqtt.tls.enabled:
-            self.__mqtt_client.tls_set(
-                ca_certs=config.mqtt.tls.ca_cert_path,
-                certfile=config.mqtt.tls.client_cert_path,
-                keyfile=config.mqtt.tls.client_key_path
-            )
+            if config.mqtt.tls.insecure:
+                self.__mqtt_client.tls_set(cert_reqs=ssl.CERT_NONE)
+                self.__mqtt_client.tls_insecure_set(True)
+            else:
+                self.__mqtt_client.tls_set(
+                    ca_certs=config.mqtt.tls.ca_cert_path,
+                    certfile=config.mqtt.tls.client_cert_path,
+                    keyfile=config.mqtt.tls.client_key_path
+                )
```

Comment:
Would you mind to add info log messages telling what tls configuration is being applied?

Reply:
Added log messages